apps/web/src/app/(prompt)/edit/[agentId]/[postGroupId]/detail/_components/EditPost/EditPost.tsx

```diff
@@ -83,13 +84,25 @@ export function EditPost() {
       confirmButton: '삭제하기',
       cancelButton: '취소',
       confirmButtonProps: {
-        onClick: () => {
-          deletePost(Number(postId));
+        onClick: async () => {
+          try {
+            await deletePost(Number(postId));
+            router.push(ROUTES.CREATE);
+          } catch (error) {
+            toast.error('삭제하지 못했어요.');
+          }
```

Comment:
```suggestion
        onClick: async () => {
          await deletePost(Number(postId), {
            onSuccess: () => router.push(ROUTES.CREATE),
            onError: () => toast.error('삭제하지 못했어요.')
          });
        },
```

p1: 이렇게 처리하면 되지 않을까요? tanstack-query에서 콜백을 제공하는데, 이를 활용하면 좋을 것 같아요.

Reply:
에러 상태의 경우, 이미 백엔드 서버로부터 받는 error 객체에서 message를 까서 해당 useMutation 내부에서 처리하고 있어요!